test/kata5.reachDestination.test.js

```diff
@@ -1,5 +1,7 @@
-const { reachDestination } = require("../src");
+const { reachDestination } = require('../src');
 
-describe("reachDestination", () => {
-  test("returns string with estimated time of arrival", () => {});
+describe('reachDestination', () => {
+	test('returns string with estimated time of arrival', () => {
+		expect(reachDestination(44, 10)).toBe('I should be there in 4.5 hours');
+	});
```

Comment:
You should add another test for the edge case when arrival time is 1, to check that the output is as expected, e.g.
```
test("returns string with estimated time of arrival when arrival time is 1", () => {
    expect(reachDestination(1, 1)).toBe("I should be there in 1 hour");
  });
```